packages/runner-ct/src/iframe/iframes.scss

```diff
@@ -16,4 +16,5 @@
 .size-container {
   overflow: auto;
   box-shadow: shadow(m);
+  max-width: 100%;
```

Comment:
Why do we want to limit against the `viewportWidth`? Can't this lead to situations where `viewportWidth` is larger than the window size, and the iframe will end up being smaller (or is `viewportWidth` the actual current browser size, in which case I'm not sure why this style is necessary)?

Reply:
Good question. This might *not* be the right behavior. I added this to match the existing behavior in both the e2e  runner and fix the regression reported by [this comment](https://github.com/cypress-io/cypress/issues/16478#issuecomment-842423403).

I pulled his project and ran it. It looks like this in my browser.

Normal viewport:

![image](https://user-images.githubusercontent.com/19196536/118745032-be2b9880-b898-11eb-82d2-8b4f642da15e.png)

Narrow viewport:

![image](https://user-images.githubusercontent.com/19196536/118745055-c8e62d80-b898-11eb-9db9-9decc75cf92f.png)

For reference, in CT runner it looks like this:

![image](https://user-images.githubusercontent.com/19196536/118745095-db606700-b898-11eb-8a48-d7bc66f3d424.png)

Without `max-width: 100%` we get the following screenshot. This just seems wrong (and a regression?)

![image](https://user-images.githubusercontent.com/19196536/118745111-e1eede80-b898-11eb-8a27-68235a0cf870.png)

With `max-width: 100%`:

![image](https://user-images.githubusercontent.com/19196536/118745180-0054da00-b899-11eb-86ce-eb16dc189931.png)